src/lesson-5-1-demo/DemoClientSidePagination.tsx

```diff
@@ -64,26 +74,30 @@ export const UsersListWithSorting: React.FC<UsersListProps> = ({
   const [sortDirection, setSortDirection] = useState<SortDirection>("ASC");
 
   const handlePreviousClick = () => {
-    // TODO
+    setCursor(cursor - pageSize);
   };
   const handleNextClick = () => {
-    //TODO
+    setCursor(cursor + pageSize);
   };
   const handleSortByChange = (event: ChangeEvent<HTMLSelectElement>) => {
-    // TODO
+    setSortBy(event.target.value as SortBy);
```

Comment:
to handle sorting, we need to manage the field to sort by and the direction of the sort